package.json

```diff
@@ -161,6 +161,7 @@
     "stickyfill": "^1.1.1",
     "style-loader": "^3.3.1",
     "swagger2openapi": "^7.0.6",
+    "update-notifier": "^6.0.2",
```

Comment:
Why we should add this package to `redoc`? We use `update-notifier` in `redoc-cli`.